app/src/androidTest/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPluginTest.kt

```diff
@@ -284,13 +299,13 @@ class LocationLayerPluginTest {
 
         plugin.forceLocationUpdate(location)
         uiController.loopMainThreadForAtLeast(MAP_RENDER_DELAY)
-        val point: Point = mapboxMap.queryLocationSourceFeatures(LOCATION_SOURCE)[0].geometry() as Point
-        Assert.assertEquals(point.latitude(), location.latitude, 0.1)
-        Assert.assertEquals(point.longitude(), location.longitude, 0.1)
+        val point: Point = mapboxMap.querySourceFeatures(LOCATION_SOURCE)[0].geometry() as Point
+        assertEquals(point.latitude(), location.latitude, 0.1)
+        assertEquals(point.longitude(), location.longitude, 0.1)
 
         plugin.isLocationLayerEnabled = false
         uiController.loopMainThreadForAtLeast(MAP_RENDER_DELAY)
-        assertThat(mapboxMap.queryLocationSourceFeatures(LOCATION_SOURCE).isEmpty(), `is`(true))
+        assertThat(mapboxMap.querySourceFeatures(LOCATION_SOURCE).isEmpty(), `is`(true))
```

Comment:
is this equivalent to `assertTrue`?

Reply:
Same as https://github.com/mapbox/mapbox-plugins-android/pull/568#discussion_r200059794.